v21.2/restore.md

```diff
@@ -168,6 +169,28 @@ The `RESTORE` process minimizes its impact to the cluster's performance by distr
 When a `RESTORE` fails or is canceled, partially restored data is properly cleaned up. This can have a minor, temporary impact on cluster performance.
 {{site.data.alerts.end}}
 
+## Restoring to multi-region databases
+
+<span class="version-tag">New in v21.2:</span> Restoring to a multi-region database is supported with some limitations. This section outlines details and settings that should be considered when restoring into multi-region databases.
+
+* A cluster's regions will be checked before a restore. Mismatched regions between [backup](known-limitations.html#backup-of-multi-region-tables) and restore clusters will be flagged before the restore begins, which allows for a decision between updating the [cluster localities](cockroach-start.html#locality) or restoring with the [`skip_localities_check`](#skip-localities-check) option to continue with the restore regardless.
+
+* A database that is restored with the `sql.defaults.primary_region` [cluster setting](cluster-settings.html) will have the `PRIMARY REGION` from this cluster setting assigned to the target database.
```

Comment:
'PRIMARY REGION' could link to https://www.cockroachlabs.com/docs/v21.2/set-primary-region